packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -296,7 +297,8 @@ storiesOf('ListView/Actions', module)
           </Item>
         </ActionMenu>
       </>
-    ), args));
+    ), args))
+  .add('Restore focus after item removal', () => <FocusExample />);
```

Comment:
```suggestion
  .add('Restore focus after item removal', (args) => <FocusExample {...args} />);
```
So we can test the difference in behavior between highlight and non-highlight selection